Makefile

```diff
@@ -24,6 +24,10 @@ AUTHORS: .mailmap .git/HEAD
 version/version.go:
 	./version/version.sh > $@
 
+${PREFIX}/bin/garbagecollect: version/version.go $(shell find . -type f -name '*.go')
```

Comment:
Don't create another binary. Implement as `registry gc`. We should already have `registry serve`, which is aliased by simply `registry`.